### DISCUSSION TITLE
Awesome-templ repo

### DISCUSSION THREAD

**Discussion by Oudwins** [a-h/templ#925] [General]
Hello everyone!

I am building a few projects using templ and have found that, although there are quite a few go packages that go well with templ, they are hard to find.

I am thinking of building an awesome-templ repo in the spirit of awesome-go. The idea is to highlight packages that work well with templ, are templ component/component libraries or have explicit integrations.

What do you all think? Any packages you would like to nominate?

Below I'll update first comment with the list of packages I have found so far

**Comment by Oudwins** [comment]
> These are just rough notes. I'll need to look a little more in depth at each package and also write a quick description

### SEO
- [teseo](https://github.com/indaco/teseo/)

### UI Frameworks
- https://hanariu.dev/
- https://github.com/iamajoe/templui
- https://www.caesar.rocks/
### UI Components
- https://github.com/indaco/goaster
- https://github.com/indaco/gropdown
- https://github.com/indaco/tabber

### Generation
- https://github.com/nokacper24/static-templ
### Web Dev Environment
- https://github.com/romshark/templier/tree/v0.3.0
### Web Frameworks
- https://github.com/go-fuego/fuego
### CSS
- [tailwind-merge-go](https://github.com/Oudwins/tailwind-merge-go)
### Validation
- [Zog](https://github.com/Oudwins/zog)

### i18n
- [ctxi18n](https://github.com/invopop/ctxi18n)

**Reply by a-h** [reply]
Brilliant. I was talking about this with Joe quite recently!

Would you be up for maintaining the repo in the templ-go org?

**Reply by Oudwins** [reply]
Sure! I'm happy to. I can commit 100% to maintaining it for at least a year. It will probably be fine even afterwards but I don't know what my life will be like in a year. If that is fine with you I'm happy to do it!

**Reply by joerdav** [reply]
Absolutely, you won't be on your own either :)

**Reply by Oudwins** [reply]
Awesome. Yea I am happy to do it. I was a little worried the message might of sounded like I was going to be unavailable after a year. Its more of I know 100% I will have the time for at least the next year but being realistic I don't think is probable that I'll be so busy as to have to stop and if it comes to that point I am sure we'll be able to find someone to replace me. I already maintain `tailwind-merge-go` and `zog` so I don't really plan on going anywhere XD

**Comment by Riu** [comment]
Good idea! Well, now is time for "templ jobs" somewhere ;)

**Comment by a-h** [comment]
OK, the repo is there.

I've granted @Oudwins collaborator level access to be able to push changes. Looking forward to it!

When we were getting templ into awesome-go, there was a set of quality criteria to meet, probably worth setting some sort of policy for this too, so that we strike the balance of being inclusive, while also making sure that the repo is really linking to awesome stuff. 😁

**Reply by Oudwins** [reply]
Perfect! I agree that de need some criteria. De can probably take inspiration from how the awesome-go repo does things. I do think that it makes sense to be a little more permissive at this early stage. At the moment most projects around templ are still quite early. We will have to trim down every so often anyway.
